lib/widgets/recent_dm_conversations.dart

```diff
@@ -138,3 +149,55 @@ class RecentDmConversationsItem extends StatelessWidget {
           ]))));
   }
 }
+
+class _NewDmButton extends StatefulWidget {
+  final VoidCallback onPressed;
```

Comment:
nit: inline the callback in the `_NewDmButtonState.build` so that we can remove this parameter; it seems fitting that this widget is responsible for calling `showNewDmSheet`